config/i18n.json

```
@@ -9,7 +9,13 @@
     "PlayStation": "PS4",
     "Stadia": "Stadia",
     "Steam": "Steam",
-    "Xbox": "XB1"
+    "Xbox": "XB1",
+    "ErrorLoading": "Unable to load Destiny Accounts from Bungie.net",
+    "MissingTitle": "Account Not Found",
+    "MissingDescription": "The account you're trying to view is not an account linked to your Bungie.net profile.",
+    "UnlinkTwitch": "Linking your Twitch account to Bungie.net will cause your D1 accounts to stop working.",
+    "UnlinkTwitchButton": "Unlink Twitch",
+    "ErrorLoadInventory": "Unable to load Destiny profile"
```

```suggestion
    "ErrorLoadInventory": "Unable to load your Destiny profile"
```
"profile" is vague to intermediate level users